roles/openshift_common/meta/main.yml

```diff
@@ -13,4 +13,4 @@ galaxy_info:
   - cloud
 dependencies:
 - role: openshift_facts
-- role: openshift_repos
+- role: openshift_docker
```

Comment:
Hmm, I'm not seeing how we can avoid the dep. Even if I restore things back the way they were in common, openshift_common requires the openshift_version variable to know which package version to install, which means we must depend on the new openshift_version role, which depends on docker. 

Previously openshift_common did it's own assumptions about what openshift_version to use, which I am trying to avoid and use the role to calculate it.

I guess I could make the role dependencies conditional, so openshift_version would depend on openshift_docker_facts and docker roles only if it's a containerized install.